sources/packages/backend/apps/api/src/services/application-offering-change-request/application-offering-change-request.service.ts

```diff
@@ -30,50 +61,60 @@ export class ApplicationOfferingChangeRequestService {
    */
   async getEligibleApplications(
     locationId: number,
-    paginationOptions: PaginationOptions,
-  ): Promise<PaginatedResults<Application>> {
+    options: {
+      pagination?: PaginationOptions;
+      applicationId?: number;
+    },
+  ): Promise<PaginatedResults<Application> | Application> {
     const applicationQuery = this.applicationRepo
       .createQueryBuilder("application")
       .select([
         "application.applicationNumber",
         "application.id",
+        "programYear.id",
         "currentAssessment.id",
+        "educationProgram.id",
+        "offering.id",
         "offering.studyStartDate",
         "offering.studyEndDate",
+        "offering.offeringIntensity",
         "student.id",
         "user.firstName",
         "user.lastName",
       ])
+      .innerJoin("application.programYear", "programYear")
       .innerJoin("application.currentAssessment", "currentAssessment")
       .innerJoin("currentAssessment.offering", "offering")
+      .innerJoin("offering.educationProgram", "educationProgram")
       .leftJoin(
         "application.applicationOfferingChangeRequest",
         "applicationOfferingChangeRequest",
+        "applicationOfferingChangeRequest.applicationOfferingChangeRequestStatus IN (:...status)",
```

Comment:
I can be totally wrong. Removing the qb orwhere here puts me the idea of trying typeorm query instead of query builder . I see that we use a condition based left join here. But seeing that we can use or operator   
![image](https://github.com/bcgov/SIMS/assets/54600590/7b85eac5-c6be-4c76-b93d-137a75ded19c)
makes me think we can try going to typeorm query may be later.